src/main/webapp/view.js

```diff
@@ -15,22 +15,47 @@
 const PARAM_ID = 'id';
 const PARAM_VIDEO_ID = 'video-id';
 
+/** Timestamp that video, disccussion, and transcript were synced to. */
+let lastSyncedTime;
+
 /* exported LECTURE_ID */
 window.LECTURE_ID = getLectureId();
 window.VIDEO_ID = getVideoId();
 
 initialize();
 
 /**
- * Initializes the video player and discussion section
+ * Initializes the timer, video player, and discussion section
  * for the lecture view page.
  */
 async function initialize() {
   window.loadVideoApi();
-  window.intializeDiscussion();
+  window.loadDiscussion();
+  // TODO: Adjust time interval.
+  window.timer = window.setInterval(() => {
+    sync(window.videoPlayer.getCurrentTime(), false);
+  }, /* ms */ 1000);
   // TODO: Initialize the trancript section.
 }
 
+
+/**
+ * Calls functions that seek video (if {@code syncVideo} is true),
+ * transcript, and discussion to {@code currentTime}.
```

Comment:
Could you specify what the units of `currentTime` are?  A number representing the total number of seconds since the start of the video?